modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -3,88 +3,614 @@
+
+It set outs the {productname} 6.0 changes that customers using TinyMCE 5 should be aware of as part of upgrading.
+
+NOTE: For support related to migration, please contact https://support.tiny.cloud/hc/en-us/requests/new[Tiny support].  +
+Open Source users: please report issues in the https://github.com/tinymce/tinymce/[TinyMCE GitHub Repository].
 
 == Contents
 
-* xref:editor-core[Editor-Core]
-* xref:plugin-changes[Plugin Changes]
+* xref:apis[APIs]
+* xref:commands-and-queries[Commands and queries]
+* xref:options[Options]
+* xref:text-patterns[Text patterns]
+* xref:plugins[Plugins]
+* xref:schema[Schema]
+* xref:ui-and-ux-elements-and-component[UI and UX elements and components]
+
+
+include::6.0-release-notes-core-changes.adoc[]
+
+[[apis]]
+== APIs
+
+=== AddOnManager
+
+Previously, when loading a requested resource, the `AddOnManager` passed a `success` or `failure` callback to report the status of the request. In {productname} 6, `AddOnManager` reports this status by returning a `Promise`, with a value of either `resolve` or `reject`.
+
+=== `closeButton` and `timeout`
+
+Previously, using `closeButton` to disable the close button presented as part of a _Notification_ required a `timeout` value to also be set.
+
+In {productname} 6, `closeButton` shows or hides the close button regardless of a `timeout` value being set or not set.
+
+Equivalently, setting a `timeout` value sets how long the notification presents before automatically closing, regardless of whether a close button is displayed or not.
+
+NOTE: This does not change the behaviour of existing _Notifications_. An `editor.notificationManager` with both `timeout: 5000` and `closeButton: false` set will still present a notification without a close button that automatically closes after displaying for 5 seconds.
+
+=== `disabled` functions renamed in all {productname} APIs
+
+Previously, multiple {productname} APIs and UI classes used `disable` or `disabled` in their function or property names.
+
+In {productname} 6, these function names have been changed and configuration changes are required.
+
+. `isDisabled()` functions are now `isEnabled()` functions.
++
+Configurations that called `isDisabled()` must now call `isEnabled()`. Also, the returned or set value must be interpreted in terms of the new function name. For example, if `isDisabled(false)` was previously called, the equivalent new call is `isEnabled(true)`.
++
+. The `enable()` and `disable()` functions are now the single `setEnabled(<state>)` function.
++
+Rather than calling the function to set the desired state, call `setEnabled()` with the `true` or `false` value to set the desired state.
++
+. The `disable(<name>)` and `enable(<name>)` functions are now the single `setEnabled(<name>, <state>)` function in the Dialog APIs.
++
+As well, the `.disabled` property is now the `.enabled` property in all {productname} UI classes.
+
+=== `editor.SetContent`
```

Comment:
`editor.SetContent` doesn't exist, so this should probably be something like this? Though it would also need reordering then.
```suggestion
=== `SetContent` event
```